src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -161,6 +183,27 @@ public void TestPushDocContinuous()
             }
         }
 
+        [Fact]
+        public void TestPullDocWithFilter()
+        {
+            using (var doc1 = new MutableDocument("doc1")) {
```

Comment:
Same as the pushdocwithfilter comment for this one, except the reverse (insert into `_otherDB` and check `Db`)

Reply:
updated.